project/src/main/webapp/candidate.html

```diff
@@ -21,6 +21,35 @@ <h2>Voter Central Candidate Page</h2>
       <div class="candidate-page-info" id="news-articles-container">
       </div>
       <div class="candidate-page-info" id="social-media-container">
+        <div id="official-feed">
+          <h4>Official Twitter Feed</h4>
+          <img src="/images/officialtweet1.png" alt="Two years ago today, I went
+              from waitress and organizer to winning a Congressional primary
+              while outspent 10-1. Every day since I’ve tried to do right by
+              working families & the movement that made it happen. Corrupt
+              interests haven’t had a good night’s sleep since Emoji:winking face
+              Thank you Emoji:purple heart"/>
+          <img src="/images/officialtweet1.png" alt="Our team has been working
+              with artists to reimagine communities into modern day, WPA-style
+              Green New Deal posters. So far we’ve got the Bronx, Queens,
+              Detroit, Boston, Los Angeles, & San Juan Emoji:Flag of Puerto Rico
+              You can find them at https://shop.ocasiocortez.com"/>
+        </div>
+        <div id="mentions-feed">
```

Comment:
Could be nice if this was side-by-side/inline horizontally with the "Official Twitter Feed"

Reply:
I think that's the plan eventually. I will try but it will probably take a bit of work to make it happen.

Reply:
Kk, no need to do it now if it'll take too long (esp if you're planning to not show "mentions" until you can figure a way to moderate it somehow)